qbraid/programs/typer.py

```diff
@@ -145,14 +127,14 @@ def get_qasm_type_alias(qasm: str) -> str:
         str: The QASM version alias ('qasm2' or 'qasm3').
 
     Raises:
-        QasmError: If the string does not represent a valid OpenQASM program.
+        QbraidQasmError: If the string does not represent a valid OpenQASM program.
```

Comment:
Since above we're now importing `QasmError` under this `QbraidQasmError` alias, can you make sure that in readthedocs, sphinx is correctly symlinking to the original `QasmError` object from this docstring?

Reply:
We do not include the [`get_qasm_type_alias` in our docs](https://github.com/qBraid/qBraid/blob/9b951e29f3a79470c6fd7fe88efa9677698bee8c/qbraid/programs/__init__.py#L35), do we want to add that?

Reply:
But I checked and sphinx does not correctly symlink to the original error object, its only the docstring that it refers to

Reply:
Yes, using - 
```
from .exceptions import QasmError
```
raising `QasmError` and using `QasmError` in the docstring correctly symlinks to the exception.

Reply:
Added a note which contains the link to the parent error